cis194/week8/zehua/src/Party.hs

```diff
@@ -0,0 +1,52 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+module Party where
+
+import           Data.List (sort)
+import           Data.Tree (Tree (Node, rootLabel, subForest))
+import           Employee
+
+-- ex1
+glCons :: Employee -> GuestList -> GuestList
+glCons e gl = GL (e:el) (f + glf)
+  where
+    Emp { empFun = f } = e
+    GL el glf = gl
```

Comment:
not sure which one is better: inline pattern matching or doing it in `where`.

Reply:
Inline pattern match is used more when pattern match on constructors